### PR TITLE
Adding Translation layer logs

### PR DESCRIPTION
**Pull request by vagisha-nidhi**
Adding some logs in translation layer

### CODE REVIEW

File: src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs [hunk 1]
```diff
@@ -132,12 +136,14 @@ public async Task<int> InitializeCommunicationAsync(int clientConnectionTimeout)
         /// <inheritdoc/>
         public void InitializeExtensions(IEnumerable<string> pathToAdditionalExtensions)
         {
+            EqtTrace.Info("VsTestConsoleRequestSender.InitializeExtensions: Initializing extensions.");
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs]: Shall we log the information as well ? Like pathToAdditionalExtensions

File: src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleWrapper.cs [hunk 2]
```diff
@@ -138,8 +140,9 @@ public void StartSession()
 
         /// <inheritdoc/>
         public void InitializeExtensions(IEnumerable<string> pathToAdditionalExtensions)
-        {
+        {
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleWrapper.cs]: Nit: remove

File: src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs [hunk 3]
```diff
@@ -107,6 +109,8 @@ public bool WaitForRequestHandlerConnection(int clientConnectionTimeout)
         /// <inheritdoc/>
         public async Task<int> InitializeCommunicationAsync(int clientConnectionTimeout)
         {
+            EqtTrace.Info("VsTestConsoleRequestSender.InitializeCommunicationAsync: Initializing Communication with vstest.console.exe.");
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs]: Message, Say it started, and probably add ended at the end of this api

Comment (vagisha-nidhi) [reply src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs]: Done

File: src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs (hunk 1 above)
Comment (vagisha-nidhi) [reply src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs]: Added

**Review by singhsarab — APPROVED** [review]

File: src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs [hunk 4]
```diff
@@ -126,18 +132,22 @@ public async Task<int> InitializeCommunicationAsync(int clientConnectionTimeout)
...
 
+            EqtTrace.Info("VsTestConsoleRequestSender.InitializeCommunicationAsync: Ended.");
+
             return this.handShakeSuccessful ? port : -1;
         }
 
         /// <inheritdoc/>
         public void InitializeExtensions(IEnumerable<string> pathToAdditionalExtensions)
         {
+            EqtTrace.Info($"VsTestConsoleRequestSender.InitializeExtensions: Initializing extensions with additional extensions path {string.Join(",", pathToAdditionalExtensions.ToArray())}.");
```
Comment (singhsarab) [inline src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs]: Add an if check for EqtTrace Info enabled

Comment (singhsarab) [reply src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs]: Also the build failed for linux

**Pull request merged**